components/accordion/src/index.scss

```diff
@@ -1,32 +1,88 @@
-/* accordion component specific classes */
-cagov-accordion .cagov-accordion-card {
-	border-radius: 0.3rem !important;
+/* v3 styles: new section */
+cagov-accordion details {
+  border-radius: 0.3rem !important;
 	margin-bottom: 0;
 	min-height: 3rem;
 	margin-top: 0.5rem;
 	border: solid 1px #ededef !important;
 }
 
-cagov-accordion .accordion-card-container {
-	display: block;
-	overflow: hidden;
+cagov-accordion:defined details {
+  transition: height .2s;
+  height: 3rem;
+  overflow: hidden;
+}
+
+/* let it be open initially if details has open attribute */
+cagov-accordion:defined details[open] {
+  height: auto;
 }
 
-cagov-accordion button.accordion-card-header {
-	display: flex;
-	justify-content: left;
-	align-items: center;
+cagov-accordion:defined summary::-webkit-details-marker {display: none; }
+
+details summary {
+  cursor: pointer;
 	padding: 0 0 0 1rem;
-	background-clip: border-box;
 	background-color: #ededef;
-	border: none;
 	position: relative;
-	width: 100%;
 	line-height: 3rem;
+  margin: 0;
+  
+  color: var(--primary-color, #064e66);
+  font-size: 1.125rem;
+  font-weight: bold;
+}
+
+cagov-accordion:defined details summary {
+  list-style: none; /* hide default expansion triangle after js executes */
+  border-radius: 5px 5px 0 0;
```

Comment:
Do we have a border-radius css variable yet? if not we should open an issue for it.

Reply:
Thank you!

Reply:
Using border-radius css var now